bin/martinize2

```diff
@@ -686,9 +686,17 @@ def entry():
         LOGGER.debug("Writing molecule {}.", molecule, type='step')
 
     # Write the topology if requested
+    # grompp has a limit in the number of character it can read per line
+    # (due to the size limit of a buffer somewhere in its implementation).
+    # The command line can be longer than this limit and therefore
+    # prevent grompp from reading the topology.
+    gromacs_char_limit = 4000  # the limit is actually 4095, but I play safe
+    command = ' '.join(sys.argv)
+    if len(command) > gromacs_char_limit:
+        command = command[:gromacs_char_limit] + ' ...'
```

Comment:
Set it to 80, and linebreak properly. Keep printing the complete command, just linewrap it.
Also see: https://docs.python.org/3/library/textwrap.html

Reply:
I considered that, but:
1. it is a fairly rare corner case so it should not affect the general case
2. the line, once wrapped, will make a header that will be very long and push the relevant information (the topology) far bellow
3. the line can be easily copy-pasted, which is not the case if it is wrapped because the comment character will pollute the text block; I am fine with if a pathologic line cannot be copied, but I'd rather not affect the legit ones.

Reply:
Good arguments. Approved.